modules/getting-started/pages/build-a-hello-world-application.adoc

```diff
@@ -27,7 +28,7 @@ image::blank-canvas.png[blank+canvas]
 +
 image::add-http.png[add+http]
 +
-Studio automatically wraps the connector with a flow, thus saving you the step of manually creating one. There will be a red error dot overlaid on the HTTP connector because you have not configured it yet.
+Studio automatically wraps the connector with a flow, so you don't have to manually create one. There will be a red error dot overlaid on the HTTP Listener because you have not configured it yet.
```

Comment:
A red error dot overlays the HTTP... (to get rid of passive construction & "will"